ext/intl/calendar/calendar_class.cpp

```diff
@@ -307,6 +307,9 @@ void calendar_register_IntlCalendar_class(void)
 	CALENDAR_DECL_LONG_CONST("FIELD_JULIAN_DAY",			UCAL_JULIAN_DAY);
 	CALENDAR_DECL_LONG_CONST("FIELD_MILLISECONDS_IN_DAY",	UCAL_MILLISECONDS_IN_DAY);
 	CALENDAR_DECL_LONG_CONST("FIELD_IS_LEAP_MONTH",			UCAL_IS_LEAP_MONTH);
+#if U_ICU_VERSION_MAJOR_NUM >= 73 && !defined(U_HIDE_DRAFT_API)
+	CALENDAR_DECL_LONG_CONST("FIELD_ORDINAL_MONTH",			UCAL_ORDINAL_MONTH);
```

Comment:
maybe this constant still makes sense?

Reply:
Once it becomes stable, sure. We're not using other draft API AFAICT.